control-plane/stor-port/src/types/v0/store/snapshots/volume.rs

```diff
@@ -87,9 +116,11 @@ impl PartialEq for VolumeSnapshotCreateInfo {
     }
 }
 
+/// The replica snapshot results from the creation operation.
 #[derive(Debug, Clone, PartialEq)]
 pub struct VolumeSnapshotCreateResult {
-    replicas: Vec<ReplicaSnapshot>,
+    /// The resulting replicas including their success status.
+    pub replicas: Vec<ReplicaSnapshot>,
```

Comment:
FYI, The protobuf message type ReplicaSnapshot is being renamed as simply Snapshot in https://github.com/openebs/mayastor-api/pull/60